widgets/Window.lua

```diff
@@ -11,6 +11,8 @@ if not StdUi:UpgradeNeeded(module, version) then return end;
 function StdUi:Window(parent, width, height, title)
 	parent = parent or UIParent;
 	local frame = self:PanelWithTitle(parent, width, height, title);
+	frame:SetFrameStrata('DIALOG')
+	frame:SetPoint('CENTER');
```

Comment:
Set default frame strata to one that is in the foreground, and not invisible like it was before.

Reply:
This actually cannot be changed here as it is a breaking change, not every window is a DIALOG and setting it explicitly is a big no-no. Same goes for setting point.